code/game/objects/items/weapons/fishing_rod.dm

```diff
@@ -0,0 +1,130 @@
+#define TIME_TO_CATCH = 80
+
+// -----------------------------
+//         Bait Datums
+// -----------------------------
+
+// Base datum for bait_type
+// bait: The object to be used as bait
+// fish: list of fish you can catch and their probability
+//       uses the format list(list(<fish>, <absolute probability>), ...)
```

Comment:
`<fish>` is meant to be the typepath of the object you'll catch.  I don't think `list(/obj/item/weapon/reagent_containers/food/snacks/fish = 100)` is a valid way to use lists, is it?